src/ostapchenko/task5/Hittable.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#ifndef FIT_GL_HITTABLE_H
+#define FIT_GL_HITTABLE_H
+
+#include "Ray.h"
+#include <QVector4D>
+#include <utility>
+
+struct Material{
+  Material() {};
+  Material( const float sF, const float  rI, const QVector4D &a) : specularFactor(sF), refractiveIndex(rI), albedo(a){}
+  float specularFactor{};
+  float refractiveIndex;
+  QVector4D albedo;
+};
+
+struct hit_record {
```

Comment:
Давайте консистентно называть структуры в CamelCase
```suggestion
struct HitRecord {
```

Reply:
исправлено